### PR TITLE
HMPB Creator Spec

### PR DESCRIPTION
**Pull request by beausmith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,58 @@
-# hmpb-creator
-Creates hand-marked paper ballots.
+# Hand-Marked Paper Ballot Creator
+
+This is a tool to create/view/edit hand-marked paper ballots. Once ballots have been proofed, the ballots and election config can be exported as an "election ballot package" which will be imported into the [Mail Ballot Manager](https://github.com/votingworks/mail-ballot-manager)
+
+## Workflow
+
+1. Admin uploads a config file (or possibly a previously created election ballot package).
+   - PDF ballots are generated from config
```
Comment (eventualbuddha) [inline README.md:8]:
```suggestion
   - PDF ballots are generated from config.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,58 @@
...
+2. Admin previews ballots by precinct/style.
+   - Admin may edit the election config at any time. 
+   - Saving changes to the election config triggers re-generation of PDF ballots.
+3. Admin exports an election ballot package.
+
+   Election Ballot Package contains:
+     - `election.json` - VotingWorks data format for election data: contests, canidates,ballot styles, precincts, etc.
```
Comment (eventualbuddha) [inline README.md:15]:
```suggestion
     - `election.json` - VotingWorks data format for election data: contests, candidates, ballot styles, precincts, etc.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,58 @@
...
+     - approved official ballots in pdf format:
```
Comment (eventualbuddha) [inline README.md:16]:
```suggestion
     - approved official ballots in PDF format:
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,58 @@
...
+- Export Election Ballot Package
+
+## Future Features
+- Edit Contest section
+  - List Contests screen
+  - Edit Contest Screen with live preview
+
+## Open Questions
+- Do we implement anything to ensure that ballots are not edited after package is created? 
+  - possibly a `ballots.json` with PDF ballot file hashes?
```
Comment (benadida) [inline README.md:58]: hash of the election package should suffice.

**Review by benadida — APPROVED** [review]

**Review by eventualbuddha — COMMENTED** [review]
Sorry, had some comments queued up but forgot to submit.

**Pull request merged**